truss/tests/test_model_inference.py

```diff
@@ -193,6 +193,33 @@ def test_requirements_file_truss():
         assert response.json() is False
 
 
+@pytest.mark.integration
+def test_docker_server_truss():
+    with ensure_kill_all():
+        truss_root = Path(__file__).parent.parent.parent.resolve() / "truss"
+
+        truss_dir = truss_root / "test_data" / "test_docker_server_truss"
+
+        tr = TrussHandle(truss_dir)
+
+        _ = tr.docker_run(local_port=8090, detach=True, wait_for_server_ready=True)
+        truss_server_addr = "http://localhost:8090"
+        full_url = f"{truss_server_addr}/v1/models/model:predict"
+
+        try:
+            response = requests.post(full_url, json={})
+            print(f"response: {response}")
+        except requests.exceptions.ConnectionError:
+            pytest.fail("ConnectionError: Could not connect to the server")
+        assert response.status_code == 200
+        assert response.json() == {
+            "message": "Hello World",
+            "is_torch_cuda_available": False,
+            "is_env_var_passed": True,
+            "is_secret_mounted": False,
+        }  # TODO: is_secret_mounted should be True
```

Comment:
I do can see secret mounted when deploy truss in cloud, but in local container it's not. Not sure if secrets can be properly mounted in the integration test setup. @squidarth please advise.

Reply:
look at how this test mounts secrets: https://github.com/basetenlabs/truss/blob/827f47bf78df7e634846ae168aa8fde4145a7104/truss/tests/test_model_inference.py#L439. You have to set them in the same way:

```
        tr = TrussHandle(truss_dir)
        LocalConfigHandler.set_secret("secret", "secret_value")
```